the-collab-lab/tcl-25-smart-shopping-list
# Add Item prompt

## Description
This PR displays a prompt in the ListView for users to add items to their list when there are no items on the list. The prompt is a message and a button which, when clicked, redirects the user to the AddView. 


## Related Issue
Closes #8 

## Acceptance Criteria


## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

1. From your terminal pull down this branch with `git pull origin cz-ci-add-item-prompt` and check that branch out with `git checkout cz-ci-add-item-prompt`.
2. Then `npm start` to launch the app.
3. Create a new shopping list or join with an existing token.
4. If the list is empty, you should see the prompt displayed. On adding an item, the list should be displayed instead of the prompt. If all items added are deleted, the app should revert to displaying the prompt.
